test/unit/s3.test.ts

```diff
@@ -30,7 +30,7 @@ _test('S3 checks', async(t) => {
     const imageExist = await s3.downloadIfPossible('bm.wikipedia.org/static/images/project-logos/bmwiki-2x.png', 'https://bm.wikipedia.org/static/images/project-logos/bmwiki-2x.png');
     t.assert(!!imageExist, 'Image exists in S3');
     // Checking the data related to image matches
-    t.equals(imageExist.headers.Metadata.etag, '"aeff-54a391a807034"', 'Etag matches');
+    t.equals(imageExist.headers.Metadata.etag, '"a740-5a6b0464619c2"', 'Etag matches');
```

Comment:
This is really weak... and consequently has been broken only a few months after been implemented. Automated test should not have the this ETAG hardcoded. If the whole lifecycle has been tested properly you should not need to do that. Should be pretty easy to fix.

Reply:
I have removed this case, because lifecycle is getting tested.